content/source/docs/cloud/agents/index.html.md

```diff
@@ -191,6 +191,12 @@ Runs which are processed by an agent will have additional information about that
 
 You may choose to run multiple agents within your network for improved resiliency. If there are multiple agents available within an organization, Terraform Cloud will select the first available agent.
 
+#### Resilience
+
+Presently, there isn't a provision for backup or standby agents. Agents should be run in a scheduler (Kubernetes, [Nomad](https://www.nomadproject.io/), etc) for resource management purposes.
```

Comment:
```suggestion
If an agent terminates abnormally, any running jobs can be restarted where they left off by resubmitting them. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```